src/pages/getting-started/installing-livechat/android-widget/index.mdx

```diff
@@ -221,7 +224,9 @@ public boolean onError(ChatWindowErrorType errorType, int errorCode, String erro
 
 ## Clear chat session
 
-After your user signs out of the app, you might want to clear the chat session. You can do that by invoking a static method on `ChatWindowView.clearSession(Context)` from anywhere in the app. In case your `ChatWindowView` is attached in course of the log out flow, you also going to need to reload it by calling `chatWindow.reload()` after the `clearSession` code. See <a href="https://github.com/livechat/chat-window-android/blob/master/app/src/main/java/com/livechatinc/livechatwidgetexample/FullScreenWindowActivityExample.java" target="_blank" rel="noopener noreferrer">FullScreenWindowActivityExample.java</a>.
+After your user signs out of the app, you might want to clear the chat session. You can do that by invoking a static method on `ChatWindowUtils.clearSession(Context)` from anywhere in the app. In case your `ChatWindowView` is attached in course of the log out flow, you also going to need to reload it by calling `chatWindow.reload(false)` after the `clearSession` code. See <a href="https://github.com/livechat/chat-window-android/blob/master/app/src/main/java/com/livechatinc/livechatwidgetexample/FullScreenWindowActivityExample.java" target="_blank" rel="noopener noreferrer">FullScreenWindowActivityExample.java</a>.
```

Comment:
This whole section needs proofreading

Reply:
You got me - I admit that it was just `(false)` added here in one of the methods and a sentence at the end, and I assumed that the rest must be okay if it was here before. Will do it :)